gradle.properties

```diff
@@ -44,6 +44,7 @@ wildfire_gender_mod_id=3706808
 crafttweaker_version_range=[9.1.90,)
 jei_version_range=[9.7.0.180,)
 wildfire_gender_version_range=[2.9,)
+curios_version_range=[1.18.2-5.0.6.3,)
```

Comment:
This should go between the crafttweaker and JEI version ranges as this block is alphabetical